src/components/MultiLanguageCodeBlock.js

```diff
@@ -0,0 +1,170 @@
+import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
+
+/**
+ * Example Usage:
+ * 
+ * import MultiLanguageCodeBlock from "../../src/components/MultiLanguageCodeBlock";
+ * <MultiLanguageCodeBlock
+ *   curl={`curl http://localhost:3000`}
+ *   title="/token"
+ * />
+ * 
+ * Note:
+ * 
+ * curl must be correctly formatted as this component does not format
+ * it for you. 
+*/
+
+const DARK_MODE_THEME = oneDark;
+const LIGHT_MODE_THEME = oneLight;
+
+const MultiLanguageCodeBlock = ({ curl, title }) => {
+  const [response, setResponse] = useState(null);
+  const [theme, setTheme] = useState(function () {
+    if (ExecutionEnvironment.canUseDOM && window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+      return DARK_MODE_THEME;
+    } else {
+      return LIGHT_MODE_THEME;
+    }
+  }());
+
+  if (ExecutionEnvironment.canUseDOM) {
+    window.matchMedia('(prefers-color-scheme: dark)')
+      .addEventListener('change', event => {
+        if (event.matches) {
+          //dark mode
+          setTheme(DARK_MODE_THEME);
+        } else {
+          //light mode
+          setTheme(LIGHT_MODE_THEME);
+        }
+      });
+  }
+
+  useEffect(async () => {
+    const rawResponse = await fetch(
+      `https://curl-converter-backend.vercel.app/api/convert`,
+      {
+        body: JSON.stringify({
+          curl: curl,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+          accept: 'application/json',
+        },
+        method: 'POST',
+      }
+    );
+    let jsonResponse = await rawResponse.json();
+    if (rawResponse.status !== 200) {
+      console.error(jsonResponse);
+      return;
+    }
+    setResponse(jsonResponse);
+  }, []);
+
+  const CodeBlock = ({ language, title, text }) => {
+    return (
+      <div>
+        <SyntaxHighlighter
+          language={language}
+          style={theme}>
+          {title}
+        </SyntaxHighlighter>
+        <SyntaxHighlighter
+          showLineNumbers={true}
+          language={language}
+          style={theme}
+        >
+          {text}
+        </SyntaxHighlighter>
+      </div>
+    );
+  };
+
+  return <div>
+    {response === null ? <ClipLoader
+      color="#ffffff"
+      loading={true}
+      size="25px"
+      aria-label="Loading Spinner"
+      data-testid="loader"
+    /> :
+      <Tabs groupId="multi-language-platform" queryString>
+        <TabItem value="javascript" label="Curl">
+          <CodeBlock
+            language="bash"
+            title={title}
+            text={curl}
+          ></CodeBlock>
+        </TabItem>
+        <TabItem value="csharp" label="CSharp">
+          <CodeBlock
+            language="csharp"
+            title={title}
+            text={response?.csharp}
+          ></CodeBlock>
+        </TabItem>
+        <TabItem value="dart" label="Dart">
+          <CodeBlock
+            language="dart"
```

Comment:
does dart work or should this be javascript?
```suggestion
            language="javascript"
```

Reply:
It works in my editor, but not in Docusaurus. It works in GitHub, so I wonder if it is a Docusaurus limitation 🤔 

Examples:

```javascript
import { Embedded } from '@beyondidentity/bi-sdk-js';

const embedded = await Embedded.initialize();
```

```kotlin
import com.beyondidentity.embedded.sdk.EmbeddedSdk

EmbeddedSdk.init(
    app: Application,
    keyguardPrompt: (((allow: Boolean, exception: Exception?) -> Unit) -> Unit)?,
    logger: (String) -> Unit,
    biometricAskPrompt: String, /* Optional */
    allowedDomains: List<String>?, /* Optional */
)
```

```swift
import BeyondIdentityEmbedded

Embedded.initialize(
    allowedDomains: [String] = ["beyondidentity.com"],
    biometricAskPrompt: String,
    logger: ((OSLogType, String) -> Void)? = nil,
    callback: @escaping(Result<Void, BISDKError>) -> Void
)
```

```javascript
import { Embedded } from '@beyondidentity/bi-sdk-react-native';

Embedded.initialize(
  biometricAskPrompt: string,
  allowedDomains?: string[]
): Promise<Success>;
```

```dart
import 'package:bi_sdk_flutter/embeddedsdk.dart';

EmbeddedSdk.initialize(
    String biometricAskPrompt,
    List<String>? allowedDomains, /* Optional */
    Future<Function>? logger, /* Optional */
)
```

Reply:
Yes and no! We have to add these languages to the prism config. I'm going to make a PR to add dart, kotlin and swift

Reply:
oh nice! I wonder if we need to add csharp, ruby, rust, and java to that list. I haven't tried those yet but node, go, and python seem to be ok.